cristin-organization/src/main/java/no/unit/nva/cristin/organization/dto/v20230526/mapper/OrganizationFromUnitMapper.java

```diff
@@ -40,9 +39,10 @@ private Set<Organization> unitsToOrganizations(List<UnitDto> unitDtos) {
     }
 
     private Set<Organization> parentToSetOfOrganizations(UnitDto parent) {
-        var set = new HashSet<Organization>();
-        set.add(this.apply(parent));
-        return set;
+        return Optional.ofNullable(parent)
```

Comment:
This is just a fad, but I really like this syntax, even if it is probably slower than the old one. 👍

Reply:
The old syntax gave a list with a element with value null, so i fixed it and rather just returned an empty list, not a list with element null

Reply:
ok, so not just a fad :)